config/consumer_config.go

```diff
@@ -34,6 +33,10 @@ import (
 	"github.com/apache/dubbo-go/common/yaml"
 )
 
+const (
+	MaxWheelTimeSpan = 900e9 // 900s, 15 minute
```

Comment:
if u need this param, why submit a pr to getty to expose it for dubbogo?

Reply:
honestly , I don't think this is a good idea.

In this case, getty developer changed the variable scope, we don't know.

So if this variable scope is `dubbo-go` , i suggest keep it in `dubbo-go` is better.